.travis.yml

```diff
@@ -1,9 +1,12 @@
 language: python
-python:
-  - "2.7"
-  - "3.5"
+python: 2.7
+env:
+  matrix:
+    - TOXENV=py27
+    - TOXENV=py36
```

Comment:
@iham this seems to be failing while the previous approach (the lines you remove above) were working correctly, is  there any specific reason for this switch?